pkg/notification/mail.go

```diff
@@ -0,0 +1,59 @@
+package notification
+
+import (
+	"fmt"
+	"net/smtp"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/rodrigo-brito/ninjabot/pkg/model"
+)
+
+type Mail struct {
+	SMTPServerAddress string
+	Email             string
+	Password          string
+}
+
+func (t Mail) Notify(text string) {
+	err := smtp.SendMail(t.SMTPServerAddress,
+		smtp.PlainAuth("", t.Email, t.Password, domain(t.SMTPServerAddress)),
+		t.Email, []string{t.Email}, []byte(text))
```

Comment:
It will be executed in every call. Similar to [Telegram](https://github.com/rodrigo-brito/ninjabot/blob/main/pkg/notification/telegram.go#L19), the `smtp.PlainAuth` can be initialized on a constructor:

Example:
```go
func NewMailer(email string, password, smtpAddress string) Mail {
  return Mail {
      auth: smtp.PlainAuth("", t.Email, t.Password, domain(t.SMTPServerAddress)),
   }
}
```

And here it can be:
```suggestion
	err := smtp.SendMail(t.SMTPServerAddress,
		t.auth,
		t.Email, []string{t.Email}, []byte(text))
```

Also, I think that emails must follow the RFC 822 for the body message. Example:
```
From: someone@example.com
To: someone_else@example.com
Subject: An RFC 822 formatted message

Your message here
```